src/AspNet.Security.OAuth.Keycloak/AspNet.Security.OAuth.Keycloak.csproj

```
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Our dev branch now targets .NET 5.0.

```suggestion
    <TargetFramework>net5.0</TargetFramework>
```